data_validation/__main__.py

```diff
@@ -266,6 +266,14 @@ def build_config_managers_from_args(args):
         args.tables_list, default_value=[{}], is_filesystem=is_filesystem
     )
 
+    filter_status = consts.VALIDATION_STATUSES
+    if args.filter_status is not None:
+        arg_list = cli_tools.get_arg_list(args.filter_status)
+        if all(arg in filter_status for arg in arg_list):
```

Comment:
once we instantiate filter_status to None, we need to use consts here instead
```suggestion
        if all(arg in consts.VALIDATION_STATUSES for arg in arg_list):
```